### PR TITLE
Fix tests for boundary conditions in mechanics model

### PR DESCRIPTION
**Pull request by finsberg**

### CODE REVIEW

File: src/simcardems/mechanics_model.py [hunk 1]
```diff
@@ -711,6 +711,11 @@ def dirichlet_bc(W):
         if fix_right_plane:
             bcs.extend(
                 [
+                    dolfin.DirichletBC(
+                        W.sub(0).sub(0),  # u_x
+                        dolfin.Constant(0.0),
+                        right,
+                    ),
```
Comment (finsberg) [inline src/simcardems/mechanics_model.py:718]: Shouldn't this be added when we fix the right plane @IlsevanHerck @cdaversin ?

Comment (cdaversin) [reply src/simcardems/mechanics_model.py:718]: I agree, and then it is consistent with the conditions when fixing the left plane

**Review by cdaversin — APPROVED** [review]

File: src/simcardems/mechanics_model.py (hunk 1 above)
Comment (IlsevanHerck) [reply src/simcardems/mechanics_model.py:718]: Not if we use the fix_right_plane as we did before, to fix the movement in y and z. Fixing the x-direction was only set in the traction parameter (completely fixed in x --> traction=0).

Comment (IlsevanHerck) [reply src/simcardems/mechanics_model.py:718]: I agree, but we implemented it to have the same constraints on the left and right side of the mesh (except x). At this point, we don't need the `fix_right_plane` and `traction=0`, as they should do the same thing.
